package.json

```diff
@@ -14,11 +14,16 @@
         "katex": "^0.15.1"
     },
     "devDependencies": {
+        "@types/d3": "^7.1.0",
+        "@types/d3-scale": "^4.0.2",
+        "@types/katex": "^0.11.1",
+        "typescript": "^4.5.4",
         "vite": "^2.7.4"
     },
     "description": "Play eye-catching math visualization on the web using SVG",
-    "module": "src/index.js",
-    "main": "src/index.js",
+    "module": "dist/index.js",
+    "main": "dist/index.js",
```

Comment:
I included `dist/` dir in `.gitignore` file, so should we set entry point to `src/` dir? I don't clearly understand about TypeScript entry point, but I can guess that that file should be `index.d.ts` in the `src/` dir, shouldn't it?